doc: update known issues with Matter's redundant Thread traffic (#14027)
nrfconnect/sdk-nrf: doc/nrf/releases_and_maturity/known_issues.rst

The issues in this section are related to the :ref:`ug_matter` protocol.

> .. rst-class:: v2-5-0 v2-4-0 v2-3-0 v2-2-0 v2-1-0 v2-0-0 v1-9-0 v1-8-0 v1-7-0

Reviewer: You missed all minor releases here.

Reviewer 2: Also, do not mention versions pro v2.1.1 as Matter was experimental there.

> KRKNWK-18495: When dimming a light/bulb over time or changing color, the thread network can be spammed.

Reviewer: I know this is a description that origins from the ticket, but could we modify it to more clearly describe how it affects NCS (e.g. that it affects only Matter Light Bulb sample (link it), explain what actually is Thread network error).

Reviewer 3: I have to agree that this needs additional clarification. For example, I am not sure what "over time" is doing here. The interpretations I can draw, in order of most to least likely based on current wording:

* When a light bulb is set to dim or change color over a long period of time (so a very gradual change), it can end up spamming the Thread network during that time.
* When a light bulb is dimmed or its color is changed several times over a long period of time, it can start spamming the Thread network.
* Either of the two above options, but the spamming only happens when the process is started.

Reviewer 4: I changed the description. @kkasperczyk-no, @wiba-nordic  please take a look.

> KRKNWK-18495: The Color Control server's RemainingTime attribute change may be reported every 100ms, even though the color temperature value handled by the Color Control server is not modified.

Reviewer 3: ```suggestion
KRKNWK-18495: The Color Control server's ``RemainingTime`` attribute change may be reported every 100 ms, even though the color temperature value handled by the Color Control server is not modified.
```

  This behavior can lead to missed thread messages or even to thread network error.

>   **Workaround:** Manually cherry-pick and apply the commit with the fix to ``sdk-connectedhomeip`` (commit hash: ``836390ed636ca36126dbcbe763d0f127626cba8d``).

Reviewer: This hash is incorrect.

>   This may result in spamming of the Thread network with unnecessary network traffic when controlling :ref:`matter_light_bulb_sample`'s brightness or color.

Reviewer 3: ```suggestion
  This can result in the Thread network being spammed with unnecessary network traffic when controlling the brightness or color of the :ref:`matter_light_bulb_sample` sample .
```
